i18n/en/code.json

```diff
@@ -0,0 +1,289 @@
+{
+  "theme.ErrorPageContent.title": {
+    "message": "This page crashed.",
+    "description": "The title of the fallback page when the page crashed"
+  },
+  "theme.BackToTopButton.buttonAriaLabel": {
+    "message": "Scroll back to top",
+    "description": "The ARIA label for the back to top button"
+  },
+  "theme.blog.archive.title": {
+    "message": "Archive",
```

Comment:
What do you mean archive page

Reply:
The two you mentioned can be deleted directly. I used the command npm run write-translations -- --locale zh-Hans, which will automatically integrate the code.js file into i8n. If the page does not have it, you can delete it.